### ISSUE TITLE
Preparation 1 Issue

### ISSUE THREAD

**Issue by dykesb** [gvsu-sta518/community#78]
Learning during a pandemic is difficult. Teaching during a pandemic is also difficult! Share a story from the past year of a silly technology issue that you did or witnessed. I'll start things off:

I taught a Saturday, 9 am - noon 😭 Intro Stat course last Winter. During the first class session, I found out that if my partner used the microwave during this time, the internet completely disappeared in my office. I had just started to introduce an activity and then everything failed to load and I was **booted** from the session. I cried a little (a lot) while I tried to figure out what was going on.

**Comment by hchenfeng** [comment]
Sorry to hear about the network issue. You probably already know about this, but just in case, a ***powerline adapter/extender*** might help if you have an ethernet port on your computer and you don't want to manage excessively long ethernet cables. For example, one of [these](https://www.tomsguide.com/best-picks/best-powerline-extenders).

**Comment by hchenfeng** [comment]
One recent issue I have seen happened when I was trying to set up ssh access for my **RStudio** environment, the workspace provided by GV. I'm not sure if I remember this correctly, but I thought I was able to create my own key in the `~/.ssh/` directory and use it for git ssh access in past classes. However, this time when I created my own key and attempted to test the connection, it kept giving me a "Bad owner or permission" error. I verified the key files had the proper permissions for ssh. And I'm still not sure what the problem was. My suspicion is there might be some restriction on the home directory top level folders that I'm not aware of. 

I later tried creating ssh key using the RStudio menu option and that worked out without any issue. The only complaint I have is that I could **NOT** change key type or length.

**Comment by rogerlog1** [comment]
# Good Afternoon

I had a **really** annoying tech issue last year. One night I reset my PC for a windows update. Totally normal, nothing out of the ordinary. That was until the next morning when I discovered that my PC was no longer able to play audio... at all. Which is a pretty big problem when you have zoom calls for classes to attend. I tried everything from updating drivers to even **buying new hardware for my computer**! In the end, I was able to fix it by doing a fresh install of windows.

**Comment by Robert-Bilyk** [comment]
One time I was attempting to set up a server for a game my friends and I played. I was following a tutorial and doing everything it said but when it got to the end, none of my friends could see the server. I spent almost **5 HOURS** trying to fix it by:
* checking the port forwarding
* checking interference from other programs
* messing with my firewall
* messing with internet settings

It turns out my ISP was simply blocking the connection so I wasted a lot of time for nothing.

**Comment by ItsOyondi** [comment]
This is a really funny. **During the pandemic**, we were having a virtual lesson using Zoom, then the power went off, meaning my router shutdown and **The internet also was disconnected** although my computer did not. I had to wait and inquire from others about anything I might have missed. Indeed the _pandemic has impacted everyone_ in one way or another.  Suggestions for improvement on virtual meetings:

- Ensure stable power supply
- Have alternative and automated power supply systems
- New technology solutions should be implemented to automatically inform users whenever power is going off.

**Comment by Volcaetus** [comment]
My tech issues story:

I was in virtual  *WebEx* interview for a job last year due to the pandemic. 
When in the meeting, my cat 
<img width="200"  src="https://user-images.githubusercontent.com/97562591/149048542-409495ca-7910-4fe1-a73f-d6c648f2da72.jpg">
decided to start playing with and chomping on the cords inserted into my pc.
My webcam started going in and out and eventually caused me to lose my video feed.
Somehow, I moved onto the next round and it became a funny story to share.

**Comment by mikaelaberg** [comment]
This past year, I worked with a group from GVSU's AWRI who had been collecting data off  of a buoy in Muskegon Lake. I was trying to call to an API to access their historical database. This was my first time attempting a call to an API, so it took me about months of troubleshooting my code before I caved and reached out to the people at GV who made the API. Turns out, in the ***11 YEARS*** they had that API, no one checked to make sure you could call it. For anyone interested, here is the [link](https://www.gvsu.edu/wri/buoy/data-api.htm) to the site.

**Comment by jfahne** [comment]
During one of my undergraduate courses during COVID I didn't realize I had somehow turned on Zoom studio effects which in _theory_ is supposed to tune up your facial features. Instead it had me looking like this:
![carol glasses](https://media.giphy.com/media/lnUmBxU32RkFxVAdEW/giphy.gif)

**Comment by kraklaua** [comment]
I had an online class last semester with **two** professors and neither was very tech savvy so they couldn't figure out that if one professor shared their screen, the other couldn't also control it so they would bicker at each other if the slides were moving too fast or too slow.  *Lovely* people but it was a little *awkward*.

**Comment by toribasso** [comment]
Technical issues were **extremely** common last school year. As a virtual tutor for statistics, it was difficult when Blackboard Collaborate was down or people were unable to connect for various reasons. One *common* reason for technical difficulties was that a students microphone or video would not work. The tutors would have to coach them through different actions that could resolve the issue.

**Comment by funckytown1** [comment]
One of the _random_ tech issue I ran into was my mic on my laptop was not working. I tried **everything** I could think of to try and fix it and eventually realized the mute mic button on my computer had been bumped. :man_facepalming:

**Comment by hill-kemei** [comment]
One day at the height of the pandemic, I was having a zoom meeting with my colleagues which was being chaired by our boss. He hates booze and one of my colleague had a beer bottle which he had strategically put on a side of his table but he had failed to adjust the laptops webcam and it kept picking the beer bottle. At one point, actually the camera got him with the bottle in his mouth. I did send him a message on the phone but it appears that he didn't have his phone with him at that time. Our boss was boiling with furry going by the look on his face.

**Comment by skirwa** [comment]
I remember at the beginning of the pandemic I used to be **thrown out** of virtual meetings due to unstable internet connection. It was really *frustrating*, since I spent most of my time rejoining the meetings instead of actually being part of those meetings. But eventually I got a stable internet connection, which enhanced my experience.

**Comment by casirose** [comment]
# My Tech Issue

One time when I was using my laptop for work, my laptop warned me with low battery then I immediately ran to get my charger and in hurry plug it in a multi plug *but the multi plug was not connected to power*, then the laptop turned off. I though the laptop had some issue but later i figured it out that multi plug was not connected to the power socket.

**Comment by moreen19** [comment]
I had a lot of zoom meetings to attend during the pandemic and i noticed that whenever i used shared **wifi** the internet connection wasn't that good . I would loose connection every once in a while and so i tried to limit my use shared wifi during meetings as much as i could. To help solve this i
  * Ensured i had  a different source of internet 
  * Used the shared wifi only when few people were using it.

**Comment by dikshashrestha** [comment]
# Work From Home - Tech Issue Experience

As we shifted to **Work From Home/Remote Work** due to pandemic, I don't know why my internet **would always used to be slow** when I was in zoom meetings and used to have a hard time during my meetings and calls as most of the time *either* I used to freeze *or* the other person. So, the zoom calls was never in my favor!

**Comment by nhroesler** [comment]
### The Classic
I started working remote for a company in September. I made a fool of myself a week in while we were in a meeting. They were discussing something that didn't pertain to me, so I was on mute just idling. Long story short, for some reason the audio and mic switched to my **laptop** instead of my microphone (which has its own mute toggle). So, I was no longer on mute. I learned this only _after_ answering a call from my girlfriend.

**Comment by vanderns** [comment]
# Tech Issue
+ For one of my classes we had to do an online presentation
+ My group wasn't very coordinated at the time
+ In a practice our slides got messed up because we didn't know when to move onto the next one

**Comment by hodgean** [comment]
## Tech Issue
Last year one of my roommates who bought a router for the house was fairly late on one or two of the internet payments and I'd be forced to run off **hotspots** for a *few days at a time.*

**Comment by shroom552** [comment]
# User Error
To this day my favorite blooper from Grand Valley was during a math lecture. The professor was walking us through a proof, alternating between a projector and a white board, and got progressively more excited as it came to the conclusion. When the *Ah-Ha* moment had arrived they turned and circled the result! The room went silent for a brief moment as everyone realized that a wonderful projector screen was lost in pursuit of Mathematics.

**Comment by bkemboi394** [comment]
*I remember having a very important virtual interview during the pandemic via zoom and forgetting to plug in my charger. Right in the middle of the interview my computer alerted me of low battery. Since my charger was not close to me, I had to ask the interviewer to excuse so I can get a charger. In the process of fetching the charger, my computer completely powered off. When I turned it back on and reconnected to zoom, I could see the frustration in my interviewer's face*

[I felt](https://giphy.com/gifs/sad-crying-the-office-YLgIOmtIMUACY)
